Bottom: c51fca71162992b9721c6a310d694b2bc0fe44e0
Top:    207801d367791d6ffa8cb5d8a06d1049c2fb76aa
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2011-05-02 15:11:39 +0100

Refresh of hid-lesstif-fix-dropped-crossh

---

diff --git a/src/hid/lesstif/main.c b/src/hid/lesstif/main.c
index cdaa7b0..3da94a9 100644
--- a/src/hid/lesstif/main.c
+++ b/src/hid/lesstif/main.c
@@ -2532,6 +2532,7 @@ idle_proc (XtPointer dummy)
       hid_expose_callback (&lesstif_hid, &region, 0);
       draw_grid ();
       lesstif_use_mask (0);
+      show_crosshair (0); /* To keep the drawn / not drawn info correct */
       XSetFunction (display, my_gc, GXcopy);
       XCopyArea (display, main_pixmap, window, my_gc, 0, 0, view_width,
 		 view_height, 0, 0);
